src/main/scala/rocket/ICache.scala

```diff
@@ -48,12 +48,13 @@ class ICache(val icacheParams: ICacheParams, val hartid: Int)(implicit p: Parame
     name = s"Core ${hartid} ICache"))
 
   val size = icacheParams.nSets * icacheParams.nWays * icacheParams.blockBytes
+  val itimAddrSpaceSize = size - icacheParams.nSets / 2 * icacheParams.blockBytes
```

Comment:
I support this reallocation of the space associated with the un-TIM-able way, but I'm not confident in this calculation's robustness. What should happen if we start scaling up icache way count for whatever reason? You can allocate N-1 ways, right?